ral-facilities/datagateway
# include lucene data in investigation requests #508

## Description
Added `luceneData` to `fetchCount` and `fetchAllIds` in the investigation tab of the search plugin, like we do for the other entities. 

## Testing instructions
- [x] Check that adding investigations with the select all button in the search plugin only adds the investigations dispalyed in the table
  - This information isn't directly displayed in the search plugin, so use dataview or download to check what's in the cart (or check the state of `cartItems`.

## Agile board tracking
closes #508
